python/ray/serve/_private/router.py

```diff
@@ -277,14 +276,19 @@ def __init__(
         controller_handle: ActorHandle,
         deployment_name: str,
         event_loop: asyncio.BaseEventLoop = None,
+        _use_ray_streaming: bool = False,
```

Comment:
Are we planning to create an API where a single handle can call both streaming and non-streaming methods on its deployment? The prototype currently marks the handle as entirely stream or entirely non-streaming.